### ISSUE TITLE
博主训练不收敛问题，小目标voc数据集1186张（训练8:验证1:测试1），冻结0，解冻200epoch（batchsize16）,total loss,val loss变化不大，相差0.011，pre0.85,recall0.69,map0.78很低，是没训练好还是要做改进，有啥好的建议吗？

### ISSUE THREAD

**Issue by ChenMaolong** [bubbliiiing/yolov4-pytorch#328]
![1661340689638](https://user-images.githubusercontent.com/84057544/186407893-e2a65ba2-38ca-417b-8849-a0498657a72e.jpg)

**Comment by bubbliiiing** [comment]
map0.78不算很低诶，要看数据集

**Comment by ChenMaolong** [comment]
> map0.78 不是很少，数据集
大佬，可是Recall只有0.69（正确判定正例的比例）怎么提高，是再增加200epoch冻结训练轮数，还是数据集少了，还是数据集标签有标错吗？

**Comment by bubbliiiing** [comment]
还是和目标有关系，0.69不算低。需要不同模型对比

**Comment by ChenMaolong** [comment]
> 还是和目标有关系，0.69不算低。需要不同模型对比

嗯嗯，谢谢
